ssh/server.go

```diff
@@ -469,6 +482,17 @@ userAuthLoop:
 					authErr = nil
 				}
 			}
+			if errors.Is(authErr, ErrDenied) {
```

Comment:
why not move this down to line 677?

Reply:
I initially had it there, but moved it up because of something that I don't recall. Done.